openshift/hac-dev
# Replace instances of ReleaseLink with ReleasePlan

## Description

The API changed in https://github.com/redhat-appstudio/infra-deployments/pull/648. I don't know how I didn't see this earlier in the day, but for me now the overview tab fails to render, making repeated requests for ReleaseLinks which no longer exist (renamed to ReleasePlans).

## Type of change



## How to test or reproduce?

I did not test this change locally - but, you should be able to verify if you can view the overview tab with `?hacbs=true` pointed at the current app studio stage cluster.

## Browser conformance:
